airbyte-webapp/src/utils/errorStatusMessage.test.tsx

```diff
@@ -0,0 +1,35 @@
+import { createFormErrorMessage, FormError } from "./errorStatusMessage";
+
+describe("#errorStatusMessage", () => {
+  it("should return a provided error message", () => {
+    const errMsg = "test";
+    expect(createFormErrorMessage(new Error(errMsg))).toBe(errMsg);
+  });
+
+  it("should return null if no error message and no status, or status is 0", () => {
+    expect(createFormErrorMessage(new Error())).toBe(null);
+    const fakeStatusError = new FormError();
+    fakeStatusError.status = 0;
+    expect(createFormErrorMessage(fakeStatusError)).toBe(null);
+  });
+
+  it("should return a validation error message if status is 400", () => {
+    const fakeStatusError = new FormError();
+    fakeStatusError.status = 400;
+    expect(createFormErrorMessage(fakeStatusError)).toMatchInlineSnapshot(`
+      <Memo(MemoizedFormattedMessage)
+        id="form.validationError"
+      />
+    `);
+  });
+
+  it("should return a 'some error' message if status is > 0 and not 400", () => {
+    const fakeStatusError = new FormError();
+    fakeStatusError.status = 401;
+    expect(createFormErrorMessage(fakeStatusError)).toMatchInlineSnapshot(`
+      <Memo(MemoizedFormattedMessage)
+        id="form.someError"
```

Comment:
Why do we want this returning `some error` rather than the actual error?  We've run into a few places in the UI recently that we've undone places that we mask the actual error returned.  Curious if that may apply here too.

Reply:
`<FormattedMessage id="form.someError" />`
It's the translation string.

Reply:
Right, but why do we want that instead of the actual error that is returned?

Reply:
This only occurs if the error does not have a message and returns a not-400 error.
I think it's just a "the server had a problem idk" type message since it didn't give us anything.